test/integration/worldwide_corporate_information_page_test.rb

```diff
@@ -41,7 +41,8 @@ class WorldwideCorporateInformationPageTest < ActionDispatch::IntegrationTest
     setup_and_visit_content_item("worldwide_corporate_information_page")
 
     assert_has_component_organisation_logo
-    assert page.has_link? "British Embassy Manila", href: "/world/organisations/british-embassy-manila"
+    assert_has_component_title("British Embassy\nManila")
```

Comment:
I don't understand why this works with a `\n` and not a `<br>`😕

Reply:
I think that's because we're matching on `text` [here](https://github.com/alphagov/government-frontend/blob/543f08202ec4633e88f0011f93be79deeaa23d83/test/test_helper.rb#L67), rather than the raw HTML, so the line break has already been parsed.